core/models.py

```diff
@@ -96,7 +96,7 @@ class Meta:
     person = models.ForeignKey(Person, on_delete=models.PROTECT)
 
     def __str__(self):
-        return '{}'.format(self.id)
+        return '{} {}'.format(self.person.name, self.person.surname.upper())
```

Comment:
esta líne aparece inofensiva pero... puede provocar el problema de N+1 queries (ver https://github.com/jmcarp/nplusone#the-problem).

Vamos a dejarlo pasar y lo arreglaremos más adelante.